rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -46,15 +51,25 @@ class QuotingPanel {
   }
 
   updateAreaNode() {
-    let areaNode = document.getElementById("calculatedArea");
-    areaNode.innerText = `${this.totalAreaInSqFt.toFixed(0)}`;
+    const areaNode = document.getElementById("calculatedArea");
+    areaNode.innerText = `${this.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
     areaNode.parentNode.parentNode.classList.remove("hidden");
+    
+    const areaNodeInModal = document.getElementById("areaModal");
+    areaNodeInModal.innerText = `${this.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
```

Comment:
Why passing `undefined`? Shouldn't you be passing `en-CA`? Or, if you actually don't want to pass anything, why not `null`?

Reply:
Given that you're using the same formatting here and on L55, maybe you should have a method `formatWithCommas` or something like that. Furthermore, you're actually setting bot `areaNode` and `areaNodeInModal` to the _same_ formatted value, so maybe you should just store that in a temp var (`const formattedArea` for example) and set both nodes to it. 

That said, hadn't you moved the modal handling to a separate class? or is that coming in another PR?

Reply:
I got that from the documentation and didn't give it a thought, I would think leaving it as 'undefined' would be better so that it is specified by the OS of the user? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Parameters

And yes I'm pretty sure I moved the methods to separate the quote panel form the modal.